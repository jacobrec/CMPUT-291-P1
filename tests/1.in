1
navras
drowssap
1
Baby
Boy
M
2019-01-01
Saskatoon
Mommy
Mom
Daddy
Dad


2222 22st
888-029-2982




1
Baby
BoY
M
2000-02-02
Transylvania
Evil
Witch
Vlad
Impaler
1
Evil
Baby
M
2000-02-02
Transylvania
Evil
Witch
Daddy
Dad
1950-01-01
Hell


7
2
