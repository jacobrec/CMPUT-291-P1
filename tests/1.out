Please sign in to use App:
1) login
2) exit
Username: Password: Enter a command:
1) register a birth
2) register a marriage
3) renew a vehicle registration
4) process a bill of sale
5) process a payment
6) get a driver abstract
7) signout
First Name: 
Last Name: 
Gender: 
Birth Date: 
Birth Place: 
Mother's First Name: 
Mother's Last Name: 
Father's First Name: 
Father's Last Name: 
Enter details for Mommy Mom
Birth Date: 
Birth Place:
Address: 
Phone Number: 
Enter details for Daddy Dad
Birth Date:
Birth Place:
Address:
Phone Number:
Enter a command:
1) register a birth
2) register a marriage
3) renew a vehicle registration
4) process a bill of sale
5) process a payment
6) get a driver abstract
7) signout
First Name: 
Last Name: 
Gender: 
Birth Date: 
Birth Place: 
Mother's First Name: 
Mother's Last Name: 
Father's First Name: 
Father's Last Name: 
The name Baby BoY is taken, please select a different name for your child
Enter a command:
1) register a birth
2) register a marriage
3) renew a vehicle registration
4) process a bill of sale
5) process a payment
6) get a driver abstract
7) signout
First Name: 
Last Name: 
Gender: 
Birth Date: 
Birth Place: 
Mother's First Name: 
Mother's Last Name: 
Father's First Name: 
Father's Last Name: 
Enter details for Evil Witch
Birth Date: 
Birth Place: 
Address:
Phone Number:
Enter a command:
1) register a birth
2) register a marriage
3) renew a vehicle registration
4) process a bill of sale
5) process a payment
6) get a driver abstract
7) signout
Please sign in to use App:
1) login
2) exit

